lib/travis/build/templates/header.sh

```diff
@@ -12,6 +12,13 @@ ANSI_GREEN="\033[32;1m"
 ANSI_RESET="\033[0m"
 ANSI_CLEAR="\033[0K"
 
+: "${SHELL:=/bin/bash}"
+: "${TERM:=xterm-256color}"
```

Comment:
Does our web interface's ansiparser support 256 colors? If not, I think `xterm` might be better?